app/assets/javascripts/features/confirm_dialog.js

```diff
@@ -0,0 +1,32 @@
+$(function(){
+  $(document).on('click', '.confirm-dialog-wrapper .close-dialog', function(ev){
+    $(this).closest('.confirm-dialog-wrapper').fadeOut(100);
+  });
+});
+
+// Override the default confirm dialog by rails
+$.rails.allowAction = function(link){
+  if (link.data("confirm") == undefined){
+    return true;
+  }
+  $.rails.showConfirmationDialog(link);
+  return false;
+}
+
+// Display the confirmation dialog
+$.rails.showConfirmationDialog = function(link){
+  var message = link.data("confirm");
```

Comment:
Wonder can we add an optional data-confirm-title to links, if it’s present use it as the title and then data-confirm as the message. If it’s not present fallback to 'Are you sure?'